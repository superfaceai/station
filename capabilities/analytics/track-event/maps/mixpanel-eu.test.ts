import { SuperfaceTest } from '@superfaceai/testing-lib';

describe(`analytics/track-event/mixpanel-eu`, () => {
  let superface: SuperfaceTest;

  beforeEach(() => {
    superface = new SuperfaceTest();
  });

  describe('UseCase', () => {
    it('should perform successfully', async () => {
      await expect(
        superface.run({
          profile: 'analytics/track-event',
          provider: 'mixpanel-eu',
          useCase: 'TrackEvent',
          input: {
            eventName: 'stationtest',
            eventProperties: {
              user: 'user_1',
            },

            // OneSDK doesn't support credentials in bodies (this must be replaced with actual key when running against live API)
            token: 'xxx',
          },
        })
      ).resolves.toMatchSnapshot();
    });
  });
});
