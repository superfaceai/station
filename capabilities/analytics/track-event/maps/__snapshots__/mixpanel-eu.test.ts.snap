// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`analytics/track-event/mixpanel-eu UseCase should perform successfully 1`] = `
Object {
  "value": "OK",
}
`;
